### PR TITLE
[Testing] add test id's for hostDisk and PV initialization tests

### PR DESCRIPTION
**Pull request by mfranczy**
**What this PR does / why we need it**:
Test id's for hostDisk and PV initialization tests

**Release note**:

```release-note
None
```

### CODE REVIEW

File: tests/storage_test.go [hunk 1]
```diff
@@ -300,218 +300,230 @@ var _ = Describe("Storage", func() {
...
+
+						By("Checking if disk.img exists")
+						vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmi, tests.NamespaceTestDefault)
+						output, _ := tests.ExecuteCommandOnPod(
+							virtClient,
+							vmiPod,
+							vmiPod.Spec.Containers[0].Name,
+							[]string{"find", "/var/run/kubevirt-private/vmi-disks/disk0/", "-name", "disk.img", "-size", "1G"},
+						)
+						Expect(strings.Contains(output, "disk.img")).To(BeTrue())
```
Comment (stu-gott) [inline tests/storage_test.go]: Failure here will result in "false != true". Could you please add an explanation as to what failed?

File: tests/storage_test.go [hunk 2]
```diff
@@ -300,218 +300,230 @@ var _ = Describe("Storage", func() {
...
-						virtClient,
-						vmiPod,
-						vmiPod.Spec.Containers[0].Name,
-						[]string{"find", hostDiskDir, "-name", diskName},
-					)
-					Expect(strings.Contains(output, diskPath)).To(BeTrue())
+						Eventually(getStatus, 30, 1).Should(Equal(k8sv1.PodSucceeded))
+					})
+
+					It("[test_id:847]Should use existing disk image and start", func() {
```
Comment (stu-gott) [inline tests/storage_test.go]: Test ID 847 seems to lay out a different test than what's here. Please double check. I'm not saying this test is wrong, perhaps the test_id is typo-ed?

Comment (mfranczy) [reply tests/storage_test.go]: Good catch! I should add one more test for our kubevirtci and add one more in polarion to reflect this one.

File: tests/storage_test.go (hunk 1 above)
Comment (mfranczy) [reply tests/storage_test.go]: Done

File: tests/storage_test.go (hunk 2 above)
Comment (mfranczy) [reply tests/storage_test.go]: Test added, I asked @ILpinto to add one additional scenario for positive tests.

**Comment by mfranczy** [comment]
all ids are in place, pr finished from my perspective

**Comment by stu-gott** [comment]
ci test please

**Pull request merged**